modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -17,12 +17,14 @@
 	--tabs-v2-tabs-wrapper-width: initial;
 
 	--tabs-v2-height: initial;
-	--tabs-v2-overflow-x: hidden;
-	--tabs-v2-overflow-y: auto;
+	--tabs-v2-overflow-x: initial;
+	--tabs-v2-overflow-y: initial;
```

Comment:
Why did you need to change this?

Reply:
@udidol 

overflow was causing issue with the box shadow > therefore I removed it by default